cutils.c

```diff
@@ -315,6 +317,153 @@ int unicode_from_utf8(const uint8_t *p, int max_len, const uint8_t **pp)
     return c;
 }
 
+/*--- integer to string conversions --*/
+
+/* All conversion functions:
+   - require a destination array `buf` of sufficient length
+   - write the string representation at the beginning of `buf`
+   - null terminate the string
+   - return the string length
+ */
+
+#define USE_SINGLE_CASE  1  // special case single digit numbers
```

Comment:
Any reason to have these here? The last 3 are unused.

Reply:
Left over from variant testing, will remove :)